nbodykit/cosmology/power/galaxy.py

```diff
@@ -0,0 +1,128 @@
+import numpy
+from . import transfers
+from ..cosmology import Cosmology
+from .linear import LinearPower
+
+class GalaxyPower(object):
```

Comment:
Perhaps rename this to FNLGalaxyPower?

Reply:
Is there a paper that we can reference for the formula used here? Perhaps we can name the class after that paper?

Reply:
Hi, here is a paper that has all the formulas and explanations. 

https://arxiv.org/pdf/2106.13725.pdf